usecases/modulecomponents/arguments/nearText/searcher_movements.go

```diff
@@ -24,9 +24,22 @@ func newMovements[T dto.Embedding]() *movements[T] {
 	return &movements[T]{}
 }
 
+const (
+	moveToError   = "move to operations are not applicable for multivector embeddings"
```

Comment:
I would define errors instead of const strings
```
var (
	ErrMultiVectorMoveTo   = errors.New("move...")
	ErrMutlVectorMoveAway = errors.New("move...")
)
```